### DISCUSSION TITLE
能将saucenao搜图功能部分封装成一个包吗？

### DISCUSSION THREAD

**Discussion by Minori-ty** [Tsuk1ko/CQ-picfinder-robot#246] [一般讨论]
我自己写的话，老是显示read ECONNRESET，无法搜图
![image](https://user-images.githubusercontent.com/79521943/132470509-44148321-9a96-4b7c-bcfb-25b4d55c99cc.png)
![image](https://user-images.githubusercontent.com/79521943/132470746-6c5b2899-346a-4e95-9e32-ef5df85aa604.png)

**Comment by Minori-ty** [comment]
大概找到原因了，是这个网站被墙了。我有翻墙软件，如何配置代理请求？

**Reply by Tsuk1ko** [reply]
axios 文档，请
